interfaces/scroll-to-text-fragment.idl

```diff
@@ -3,10 +3,10 @@
 // (https://github.com/tidoust/reffy-reports)
 // Source: Text Fragments (https://wicg.github.io/scroll-to-text-fragment/)
 
-[Exposed=Window]
+[Exposed=Document]
```

Comment:
This is wrong, will have to go fix the spec.

Reply:
Sorry about that, I changed this recently in https://github.com/WICG/scroll-to-text-fragment/pull/134 and it looks like I misunderstood what `Exposed` does. I've changed it back to `Window` which I think is correct (this makes it accessible only from the main thread rather than workers?) and opened a PR; added you as reviewer to make sure I got it right this time.

Reply:
That's right, Window effectively means the main thread, the context where the global object is a Window.